lib/validator/v3/validator_test.go

```diff
@@ -573,7 +573,10 @@ func init() {
 
 		// (API) FelixConfiguration.
 		Entry("should accept a valid DefaultEndpointToHostAction value", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "Drop"}, true),
-		Entry("should reject an invalid DefaultEndpointToHostAction value 'drop' (lower case)", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "drop"}, false),
+		Entry("should accept a valid DefaultEndpointToHostAction value 'drop'", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "drop"}, true),
+		Entry("should accept a valid DefaultEndpointToHostAction value 'RETURN'", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "RETURN"}, true),
+		Entry("should accept a valid DefaultEndpointToHostAction value 'return'", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "return"}, true),
+		Entry("should accpet a invalid DefaultEndpointToHostAction value 'Return'", api.FelixConfigurationSpec{DefaultEndpointToHostAction: "Return"}, true),
 		Entry("should accept a valid IptablesFilterAllowAction value 'Accept'", api.FelixConfigurationSpec{IptablesFilterAllowAction: "Accept"}, true),
```

Comment:
Maybe add a few similar test cases for this since we also have some case insensitive changes to the following Iptables fields?

Reply:
I think we need to have some additional test cases on the other fields that are not `DefaultEndpointToHostAction` since that uses the `dropAcceptReturnRegex` and we want to test some fields against the `acceptReturnRegex`.